module-audio/Audio/AudioCommon.hpp

```diff
@@ -82,6 +82,12 @@ namespace audio
         Enable
     };
 
+    enum class VolumeUpdateType
+    {
+        UpdateDB,
+        SkipDBUpdate
+    };
```

Comment:
What about `SkipUpdateDB`. It looks similar to `UpdateDB`

Reply:
You're right, I'll change that :wink:

Reply:
Done!